.github/actions/install-rust/action.yml

```diff
@@ -27,3 +27,6 @@ runs:
         profile: ${{ inputs.profile }}
         components: ${{ inputs.components }}
         override: true
+    
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
```

Comment:
why?

Reply:
We removed this in other repos

Reply:
🤔  It's required by `opentelemetry-prometheus` package